### PR TITLE
seacas: limit to fmt@10

### PR DESCRIPTION
**Pull request by wdconinc**
A missing include in `seacas` (and a change in what symbols are in which headers in `fmt` v11) are causing missing definitions, see https://github.com/sandialabs/seacas/pull/477.

Rather than adding a patch (which could be done when the linked upstream PR is merged), this PR limits the version of `fmt` to v10.

### CODE REVIEW

**Review by alalazo — APPROVED** [review]
@wdconinc Feel free to merge if maintainers approve (or within a few days if you don't receive further reviews)

**Review by gsjaardema — APPROVED** [review]
LGTM.  The include changes have been applied to the SEACAS package, so at the next release we can begin supporting fmt@11:

**Pull request merged**
